project/Quiet.scala

```diff
@@ -8,26 +8,6 @@ object Quiet {
   //
   //   https://github.com/scala/scala-dev/issues/100
   def silenceScalaBinaryVersionWarning = ivyConfiguration := {
-    ivyConfiguration.value match {
```

Comment:
The warnings we're trying to suppress are still emitted by SBT 1.1, so we'll have to figure out how to port this, or otherwise have a plan to make the warning suppressable in SBT itself.

```
sbt:root> test:compile
[warn] Binary version (2.13.0-pre-SNAPSHOT) for dependency org.scala-lang#scala-library;2.13.0-pre-SNAPSHOT
[warn] 	in org.scala-lang#scala-reflect;2.13.0-pre-SNAPSHOT differs from Scala binary version in project (2.13.0-M3).
[warn] Binary version (2.13.0-pre-SNAPSHOT) for dependency org.scala-lang#scala-library;2.13.0-pre-SNAPSHOT
[warn] 	in org.scala-lang#scala-compiler;2.13.0-pre-SNAPSHOT differs from Scala binary version in project (2.13.0-M3).
[warn] Binary version (2.13.0-pre-SNAPSHOT) for dependency org.scala-lang#scala-reflect;2.13.0-pre-SNAPSHOT
[warn] 	in org.scala-lang#scala-compiler;2.13.0-pre-SNAPSHOT differs from Scala binary version in project (2.13.0-M3).
[warn] Binary version (2.13.0-pre-SNAPSHOT) for dependency org.scala-lang#scala-compiler;2.13.0-pre-SNAPSHOT
[warn] 	in org.scala-lang#scalap;2.13.0-pre-SNAPSHOT differs from Scala binary version in project (2.13.0-M3).
[warn] Binary version (2.13.0-pre-SNAPSHOT) for dependency org.scala-lang#scala-compiler;2.13.0-pre-SNAPSHOT
[warn] 	in org.scala-lang#scala-compiler-interactive;2.13.0-pre-SNAPSHOT differs from Scala binary version in project (2.13.0-M3).
[warn] Binary version (2.13.0-pre-SNAPSHOT) for dependency org.scala-lang#scala-compiler;2.13.0-pre-SNAPSHOT
[warn] 	in org.scala-lang#repl;2.13.0-pre-SNAPSHOT differs from Scala binary version in project (2.13.0-M3).
[warn] Binary version (2.13.0-pre-SNAPSHOT) for dependency org.scala-lang#scala-compiler;2.13.0-pre-SNAPSHOT
[warn] 	in org.scala-lang#scala-compiler-doc;2.13.0-pre-SNAPSHOT differs from Scala binary version in project (2.13.0-M3).
[info] Running scala.tools.docutil.ManMaker fsc, scala, scalac, scaladoc, scalap /Users/jz/code/scala-sbt-1/target/scala-dist/resource_managed/main/doc/tools /Users/jz/code/scala-sbt-1/target/scala-dist/resource_managed/main/genman
[info] Compiling 526 Scala sources and 120 Java sources to /Users/jz/code/scala-sbt-1/build/quick/classes/library ...
```